manifests/freenas-12-stable-ec2.json

```diff
@@ -0,0 +1,235 @@
+{
```

Comment:
This will be a huge pain to maintain. We need  a way to extend/use the original freenas-12-stable somehow.

Reply:
@kmoore134 can this be possible to import one manifest from another and override some keys?
```
--- manifests/freenas-12-stable.json	2019-07-18 16:00:47.102116984 +0200
+++ manifests/freenas-12-stable-ec2.json	2019-07-18 16:10:01.638388395 +0200
@@ -1,4 +1,4 @@
-{	
+{
 	"base-packages": {
 		"kernel-flags": {
 			"default": [
@@ -20,12 +20,6 @@
 		},
 		"auto-install-script": "",
 		"dist-packages": {},
-		"dist-packages-glob": {
-			"default": [
-				"os-nozfs-kernel",
-				"os-nozfs-userland"
-			]
-		},
 		"prune": {
 			"default": [
 				"METALOG",
@@ -49,20 +43,10 @@
 				"/usr/lib32"
 			]
 		},
-		"file-name": "FreeNAS-12-snapshot-x64-%%DATE%%",
+		"file-name": "FreeNAS-12-snapshot-EC2-%%DATE%%",
 		"install-script": "",
-		"iso-base-packages": {
-			"default": [
-				"os-nozfs-userland",
-				"os-nozfs-kernel"
-			]
-		},
 		"iso-packages": {
-			"default": [
-				"freenas/freenas-installer",
-				"sysutils/openzfs",
-				"textproc/jq"
-			]
+			"default": []
 		},
 		"offline-update": true,
 		"overlay": {},
@@ -233,5 +217,19 @@
 		"BUILD_AS_NON_ROOT=no",
 		"RESTRICT_NETWORKING=no"
 	],
-	"version": "1.1"
+	"version": "1.1",
+	"vm": {
+		"boot": "zfs",
+		"disk-config": "zfs-noswap-freenas",
+		"file-name": "FreeNAS-12-snapshot-EC2-%%GITHASH%%-%%DATE%%",
+		"pool-name": "freenas-boot",
+		"size": "3G",
+		"vm-base-packages": {
+			"default": [
+				"os-nozfs-userland",
+				"os-nozfs-kernel",
+				"sysutils/openzfs"
+			]
+		}
+	}
 }
```